packages/gestalt/src/PageHeader/components.js

```diff
@@ -52,14 +52,16 @@ export function PageHeaderThumbnail({ thumbnail }: {| thumbnail: Element<typeof
 export function PageHeaderBadge({
   badgeText,
   badgeTooltipText,
+  type = 'info'
 }: {|
   badgeText: string,
   badgeTooltipText?: string,
+  type?: TypeOptions
 |}): Node {
   return badgeTooltipText ? (
     <Badge
       text={badgeText}
-      type="info"
+      type={type}
       position="middle"
       tooltip={{
```

Comment:
@AlbertCarreras nitpick: why not just move the `badgeTooltipText` logic to `tooltip` prop itself?

Reply:
What do you mean @diyorbek

Reply:
Can you quickly mock it in code here?

Reply:
@AlbertCarreras I meant what if we remove the ternary operator for the return statement and just `return <Badge/>`. The ternary statement can be moved to `tooltip` prop like below. I wanted to know if there was some specific reason for doing so or if it was just a code style. Nothing more)
```suggestion
       tooltip={badgeTooltipText ? {
            accessibilityLabel: '',
            text: badgeTooltipText,
            idealDirection: 'up',
          } : undefined}
```

Reply:
Got it haha Yeah, that should work and remove some line of code. Nice catch

Reply:
@AlbertCarreras alright, if there was no specific reason to do it this way, let's leave as is for now. Because for react-conciler either way is the same. It is kinda off-topic change anyways 😄.

Reply:
Seems like good cleanup — dedupes the other props being passed to Badge so there's no chance of drift